src/test/README.md

```diff
@@ -1,3 +1,25 @@
+# Unit tests
+
+The sources in this directory are unit test cases. Boost includes a
+unit testing framework, and since bitcoin already uses Boost, it makes
+sense to simply use this framework rather than require developers to
+configure some other framework (we want as few impediments to creating
+unit tests as possible).
+
+The build system is setup to compile an executable called `test_bitcoin`
+that runs all of the unit tests. The main source file is called
+setup_common.cpp.
```

Comment:
Nit: Use `setup_common.cpp` instead of "setup_common.cpp".

Reply:
done